### PR TITLE
aria2: convert into page alias for aria2c

### LINKED ISSUE #5978: Possible duplicate: aria2c and aria2
I was looking at the tldr pages and found out that there is a possible duplicate:
1. [tldr of aria2c](https://github.com/tldr-pages/tldr/blob/main/pages/common/aria2c.md)
2. [tldr of aria2](https://github.com/tldr-pages/tldr/blob/main/pages/common/aria2.md)

While programs name is `aria2` and terminal command is `aria2c`. 

I have updated [aria2c](https://github.com/tldr-pages/tldr/pull/5914) page a week ago.

Is this duplicate pages intentional?
If Yes, I suggest to create a type of symlink of `aria2` to `aria2c` as `aria2c` is the command name and is more up to date.
If No, I suggest to delete `aria2` page.

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

Hi, due to the following issue: #5978 I have reformed aria2 just to be a pointer for aria2c

closes #5978

### CODE REVIEW

File: pages/common/aria2.md [hunk 1]
```diff
@@ -1,33 +1,7 @@
 # aria2
 
-> A lightweight multi-protocol & multi-source command-line download utility.
-> Supports HTTP, HTTPS, FTP, SFTP, BitTorrent and Metalink.
-> More information: <https://aria2.github.io/>.
+> This command is an alias of `aria2c`.
 
-- Download a web resource:
+- View documentation for the original command:
```
Comment (CleanMachine1) [inline pages/common/aria2.md:5]:
```suggestion
- View documentation for the correct command:
```
opinions?

Comment (bl-ue) [reply pages/common/aria2.md:5]: Hmm, this is tricky.  We can change the wording in alias pages depending on the nature of the alias page itself.  Perhaps `new`?

Comment (CleanMachine1) [reply pages/common/aria2.md:5]: Wont always be new, in 10 years time it will be _pretty old_

Comment (bl-ue) [reply pages/common/aria2.md:5]: Then let's keep it as `original`, how about.

Comment (CleanMachine1) [reply pages/common/aria2.md:5]: That makes it sound like aria2 was a new version, `updated` maybe?

Comment (bl-ue) [reply pages/common/aria2.md:5]: Sounds good to me.
```suggestion
- View documentation for the updated command:
```

**Comment by CleanMachine1** [comment]
This should be ready for approval now

**Review by marchersimon — APPROVED** [review]
Looks good

**Review by navarroaxel — APPROVED** [review]

**Review by bl-ue — APPROVED** [review]

**Pull request merged**